lib/markdown-pdf.js

```diff
@@ -173,7 +173,8 @@ function makePdf(inputHtml, outputPath){
 function processSrc(src){
   //make a local img src path into a "file:///absolute/path/" if it isn't already
 
-  if(url.parse(src).protocol){
+  var protocolsRegEx = /^(https?|file):$/;
+  if(protocolsRegEx.exec(url.parse(src).protocol) != null){
```

Comment:
Why `exec`? `test` is better.
In case of regex do we need to `parse` url at all?